docs/project_wheres_religion_mobile/about.md

```diff
@@ -14,14 +14,17 @@ custom_edit_url: https://github.com/oss-slu/lrda_mobile
 
 - **Source Code:** [https://github.com/oss-slu/lrda_mobile](https://github.com/oss-slu/lrda_mobile) [<img src="/img/git-alt.svg" alt="git" width="25" height="25" />](https://github.com/oss-slu/lrda_mobile)
 - **Client** Dr. Adam Park
```

Comment:
missing colon

Reply:
![Screenshot 2024-08-29 133736](https://github.com/user-attachments/assets/b2487185-fd60-4921-aa68-51dadf9beb47)
I will do that, but the current portfolio page is the same as the above. I have attached an image of it; have a look.

Reply:
![Screenshot 2024-08-29 135625](https://github.com/user-attachments/assets/c44714e7-ef10-47cc-8535-517f0512a516)